rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -167,7 +168,24 @@ error_code _sys_game_board_storage_read(vm::ptr<u8> buffer1, vm::ptr<u8> buffer2
 		return CELL_EFAULT;
 	}
 
-	memset(buffer1.get_ptr(), 0, 16);
+	be_t<u64> data[2] = { +g_cfg.sys.lv2_board_storage_high, +g_cfg.sys.lv2_board_storage_low };
+	memcpy(buffer1.get_ptr(), &data, 16);
+	*buffer2 = 0;
+
+	return CELL_OK;
+}
+
+error_code _sys_game_board_storage_write(vm::ptr<u8> buffer1, vm::ptr<u8> buffer2)
+{
+	sys_game.trace("sys_game_board_storage_write(buffer1=*0x%x, buffer2=*0x%x)", buffer1, buffer2);
+
+	if (!buffer1 || !buffer2)
+	{
+		return CELL_EFAULT;
+	}
+
+	g_cfg.sys.lv2_board_storage_high.set(*reinterpret_cast<be_t<u64>*>(buffer1.get_ptr()));
```

Comment:
1. Don't modify settings on runtime.
2. It needs to be atomic, maybe with atomic_t<v128> for example.

Reply:
Done.